src/Regex.cpp

```diff
@@ -25,6 +25,19 @@ boost::shared_ptr<RegexMatch> Regex::find(const std::string &expression) const {
     return boost::make_shared<FindRegexMatch>(regexImpl, expression);
 }
 
+namespace {
+bool isStartOfCodepoint(unsigned int i) {
```

Comment:
This is tailored to UTF-8, so please put that information somewhere, e.g. `utf8CodeUnit` instead of `i` or `isUtf8CodeUnitStartOfCodepoint`.

Reply:
Haha, in mtt-japanese-search-support, I found this even shorter test: `(i & 0xc0) != 0x80)` for the start of a code point.

Reply:
https://github.com/AbletonAppDev/cucumber-cpp/pull/8#discussion_r293476620 😎